script.module.enum34/lib/enum/enum34.egg-info/SOURCES.txt

```diff
@@ -0,0 +1,13 @@
+MANIFEST.in
```

Comment:
remove

Reply:
This file provided by the original author Ethan Furman
Will removing it break the enum package?

Reply:
no this is only needed to packages for pypi

Reply:
OK